ergo/distributions/location_scale_family.py

```diff
@@ -57,6 +58,20 @@ def sample(self):
         # FIXME (#296): This needs to be compatible with ergo sampling
         return self.odist.rvs(loc=self.loc, scale=self.scale)
 
+    def get_normalized(self, scale_min: float, scale_max: float):
```

Comment:
Why `get_normalized` and `get_denormalized` instead of `normalize` and `denormalize`? The reason I can think of is to indicate that this doesn't mutate the existing instance and instead returns a new one, but I'd prefer to do this by convention for everything and clearly indicate the cases where we do mutate instances.

Reply:
yeah the non-mutating thing was exactly what I was thinking, but happy to go with non-mutating by default. Will rename.